geopandas/base.py

```diff
@@ -231,13 +231,17 @@ def exterior(self):
 
     @property
     def interiors(self):
-        """Returns a ``GeoSeries`` of InteriorRingSequences representing the
+        """Returns a ``Series`` of InteriorRingSequences representing the
```

Comment:
I've not worked with `interiors`  before, so I was surprised at this change, but see that it is a necessity as things currently stand. However, I wonder if this shouldn't be doing something more like what the `explode` method does.

Reply:
Hmm, actually exploding is maybe too much. But maybe we could already convert the InteriorRingSequence objects to actual lists? (those objects have no methods in shapely, the only thing you can do with them is converting to list / iterating through them, I think)

Reply:
That may be. I should have been clearer in any case that the suggestion about `explode` was not intended about this PR. In general, I'd expect a method that in shapely returns a geometry of some sort to return a GeoSeries within geopandas, and was thinking about what that might look like here.

Reply:
Thank you for your opinion.
It was very helpful.

I agree with your opinion that converting to list.

Because I feel that the error against non-polygon is too severe, 
I will give a warning in case of mixed geometry column.